test/csv/interface/test_read.rb

```diff
@@ -32,6 +32,25 @@ def test_foreach
     assert_equal(@rows, rows)
   end
 
+  def test_foreach_stringio
+    s = StringIO.new
+    s << @data
```

Comment:
Can we simplify this?

```suggestion
    s = StringIO.new(@data)
```